test/integration/scheduler/framework_test.go

```diff
@@ -581,8 +582,8 @@ func TestUnreservePlugin(t *testing.T) {
 				if err = waitForPodUnschedulable(cs, pod); err != nil {
 					t.Errorf("test #%v: Didn't expected the pod to be scheduled. error: %v", i, err)
 				}
-				if unresPlugin.numUnreserveCalled == 0 || unresPlugin.numUnreserveCalled != pbdPlugin.numPrebindCalled {
-					t.Errorf("test #%v: Expected the unreserve plugin to be called %d times, was called %d times.", i, pbdPlugin.numPrebindCalled, unresPlugin.numUnreserveCalled)
+				if unresPlugin.numUnreserveCalled < 1 {
```

Comment:
Can `unresPlugin.numUnreserveCalled` be negative? If not, I'd say `if unresPlugin.numUnreserveCalled == 0` makes more sense.

Reply:
fixed @Huang-Wei